darts/models/forecasting/baselines.py

```diff
@@ -173,3 +182,6 @@ def ensemble(
                 predictions.pd_dataframe(copy=False).sum(axis=1) / len(self.models),
                 static_covariates=predictions.static_covariates,
             )
+
+    def _is_probabilistic(self) -> bool:
```

Comment:
We could add these lines to the parent `EnsembleModel`, to avoid having to redefine it for all ensemble models.

Reply:
indeed